products/idn/api/authorization.md

```diff
@@ -0,0 +1,128 @@
+
+Alternatively, `sp:scopes:default` is the least privileged scope that only grants access to endpoints that require no authorization at all, such as [list public identities](https://developer.sailpoint.com/idn/api/v3/get-public-identities).
+
+Scopes are additive, which means the final right set is the intersection of all the rights granted by the scopes assigned to an OAuth credential, excluding any rights that fall outside of the user level. Each scope added to an OAuth credential builds up the credential's permission set, incrementally increasing access to the API.  If an OAuth credential has `sp:scopes:all` granted, then any additional scope is ignored because `sp:scopes:all` already contains the complete set of rights available to the user level.  
+
+:::tip
+
+Because scopes can only grant access to APIs that are a part of the rights included in a user level, it is often easier to assign the **Admin** user level to the user first and then apply scopes to the OAuth credentials to limit the access.
+
+:::
+
+### Identifying Necessary Authorization for an Endpoint
+
+Each endpoint document specifies how to authorize with the endpoint in the **Authorization** dropdown, which is located on the right side column below the endpoint path. Selecting **Authorization** expands the dropdown menu showing the details of how to authorize with the endpoint. The following image shows the authorization details of the [List Access Profiles](https://developer.sailpoint.com/idn/api/beta/list-access-profiles) endpoint.
+
+![Authorization Dropdown](./img/authorization/authorization-dropdown.png)
+
+- **type**: The type of authorization method supported for this endpoint. SailPoint uses the [OAuth2 standard](./authentication.md#oauth-20) for all the V3/Beta APIs.
+- **flow**: One or more OAuth flows supported by the endpoint. A token only needs to be generated by one flow to be valid. Refer to [Authentication Details](./authentication.md#authentication-details) for more information about the available flows.
+- **scopes**: A list of scopes necessary to access the endpoint. A token only needs one of the scopes to authorize with the endpoint. When possible, choose the least privileged scope. Scopes ending in `read` can only retrieve data. Scopes ending in `manage` can retrieve, modify, and delete data.
+
+:::info
+
+SailPoint is working to define scopes for every endpoint, but you may encounter a scenario where you need access to an endpoint that does not yet have a scope defined. Until a scope is defined for the endpoint, you can assign `sp:scopes:all` to ensure that your credentials can access the necessary endpoints. Once all of the endpoints necessary for your use case have scopes defined, you can create a new credential with the appropriate scopes in place.
+
```

Comment:
FYI, we are working on being able to update existing PATs and API clients to change scope. But that is not ready yet.

Reply:
Do you have an ETA for the update endpoint?  If it's reasonably close, I'll change the verbiage now.  Otherwise, I'll wait until that endpoint is released.